packages/design-system/collapsible-tab-view/types.tsx

```diff
@@ -26,7 +26,7 @@ export enum RefreshTypeEnum {
 export type CollapsibleHeaderProps<T extends Route> = {
   initHeaderHeight?: number;
   renderScrollHeader: (
-    translateYValue: Animated.SharedValue<number>
+    translateYValue?: Animated.SharedValue<number>
```

Comment:
hey @Daavidaviid!
since the value is not passed on the web, I changed its type to optional.
maybe you need to make a judgment, thanks!